### DISCUSSION TITLE
FINN examples V0.0.6 remove pynq-z2 in supported boards list

### DISCUSSION THREAD

**Discussion by dibaotian** [Xilinx/finn#944] [General]
Hello,    V0.0.6 remove the  pynq-z2 in supported boards list, will the board be supported in the future?

**Comment by fpjentzsch** [comment]
Hi, I believe this just means that the prebuilt bitstreams are only supplied for the PYNQ-Z1. FINN itself still supports the PYNQ-Z2, so you should be able to set `platforms_to_build` in the build.py script to "Pynq-Z2" instead of "Pynq-Z1" to rebuild the examples for this board. You can find the instructions on how to rebuild the finn-examples here: https://github.com/Xilinx/finn-examples/blob/main/build/README.md
